flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogHiveMetadataTest.java

```diff
@@ -70,6 +77,35 @@ public void testCreateTable_StorageFormatSet() throws Exception {
 		assertFalse(StringUtils.isNullOrWhitespaceOnly(serde));
 	}
 
+	// ------ table and column stats ------
+	@Test
+	public void testAlter_GET_TableColumnStatistics() throws Exception {
```

Comment:
can we follow the naming conventions? This should probably just be `testAlterTableColumnStatistics()`